toolchain/check/testdata/interop/cpp/class/method.carbon

```diff
@@ -137,6 +139,27 @@ fn Ref(p: Cpp.HasQualifiers*) {
   p->const_ref_ref_this();
 }
 
+// --- object_param_qualifiers_overloaded.h
+
+struct HasQualifiers {
+  int* _Nonnull F() &;
+  int F() const &;
+  void F() volatile &;
+};
+
+// --- use_object_param_qualifiers_overloaded.carbon
+
+library "[[@TEST_NAME]]";
+
+import Cpp library "object_param_qualifiers_overloaded.h";
+
+fn F(v: Cpp.HasQualifiers, p: Cpp.HasQualifiers*) {
```

Comment:
```suggestion
fn G(v: Cpp.HasQualifiers, p: Cpp.HasQualifiers*) {
```

A more meaningful name would also work, as would renaming `HasQualifiers::F`. The idea is just to avoid reusing unqualified names, because that makes the SemIR somewhat easier to read.

Reply:
Done.